### PR TITLE
Update SUPPORT.md

### PR DESCRIPTION
**Pull request by mjang**
I'm using this template repo to create a future open source doc repo. In the SUPPORT.md file, I exposed the link to sales, as our products do have commercial support. That link is out of date, so:

### Proposed changes

I've changed commented link in SUPPORT.md to F5.com sales

### Checklist

Before creating a PR, run through this checklist and mark each as complete:

- [x] I have read the [contributing guidelines](/CONTRIBUTING.md).
- [x] I have signed the [F5 Contributor License Agreement (CLA)](https://github.com/f5/.github/blob/main/CLA/cla-markdown.md).
- [x] If applicable, I have added tests that prove my fix is effective or that my feature works.
- [x] If applicable, I have checked that any relevant tests pass after adding my changes.
- [x] I have updated any relevant documentation ([`README.md`](/README.md) and [`CHANGELOG.md`](/CHANGELOG.md)).

### CODE REVIEW

**Comment by mjang** [comment]
I have hereby read the F5 CLA and agree to its terms

recheck

**Comment by mjang** [comment]
recheck

File: SUPPORT.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Please see the [contributing guide](/CONTRIBUTING.md) for guidelines on how to b
 
 <!-- ## Commercial Support
 
-Commercial support for this project may be available. Please get in touch with [NGINX sales](https://www.nginx.com/contact-sales/) or check your contract details for more info! -->
+Commercial support for this project may be available. Please get in touch with [F5 (NGINX) sales](https://www.f5.com/products/get-f5/) sales or check your contract details for more info! -->
```
Comment (alessfg) [inline SUPPORT.md:37]: Can you remove the parentheses in "F5 (NGINX)"?
```suggestion
Commercial support for this project may be available. Please get in touch with [F5 NGINX sales](https://www.f5.com/products/get-f5/) sales or check your contract details for more info! -->
```

**Review by alessfg — CHANGES_REQUESTED** [review]

File: SUPPORT.md (hunk 1 above)
Comment (mjang) [reply SUPPORT.md:37]: Yup, that's also consistent with our style guides, thank you.

**Review by alessfg — APPROVED** [review]

**Pull request merged**